kokoro/ubuntu/periodic.sh

```diff
@@ -13,5 +13,23 @@ mvn -B clean install
 # https://stackoverflow.com/questions/3459928/running-a-specific-maven-plugin-goal-from-the-command-line-in-a-sub-module-of-a/26448447#26448447
 # https://stackoverflow.com/questions/11091311/maven-execjava-goal-on-a-multi-module-project
 cd dashboard
+
+# Step 1: Generate dashboards for the released BOMs with the latest DashboardMain class
+# For example: https://storage.googleapis.com/cloud-opensource-java-dashboard/1.1.0/dashboard.html
+
+# Querying https://search.maven.org/classic/#api with
+# groupId:com.google.cloud AND artifactId:libraries-bom, receiving latest (sorted) versions.
+# Note: if `rows` parameter is omitted, it only returns latest 10 items.
+SONATYPE_RESPONSE=`curl 'https://search.maven.org/solrsearch/select?q=g:%22com.google.cloud%22+AND+a:%22libraries-bom%22&core=gav&rows=1000&wt=json'`
```

Comment:
Default value for rows parameter is only 10 (example below). So explicitly specifying the value here.
Example with guava
https://search.maven.org/solrsearch/select?q=g:%22com.google.guava%22+AND+a:%22guava%22&core=gav&wt=json

Reply:
Ideally we should page this.

Reply:
YAGNI